udata/core/dataset/apiv2.py

```diff
@@ -0,0 +1,151 @@
+import logging
+
+from flask import request, url_for
+from flask_restplus.reqparse import RequestParser
+
+from udata.api import apiv2, API, fields
+
+from .models import (
+    UPDATE_FREQUENCIES, DEFAULT_FREQUENCY, DEFAULT_LICENSE
+)
+from .permissions import DatasetEditPermission
+from .api_fields import (
+    resource_fields,
+    dataset_fields,
+    badge_fields,
+    org_ref_fields,
+    user_ref_fields,
+    temporal_coverage_fields,
+    spatial_coverage_fields,
+    DEFAULT_MASK
+)
+
+log = logging.getLogger(__name__)
+
+ns = apiv2.namespace('datasets', 'Dataset related operations')
+resources_parser = RequestParser()
+resources_parser.add_argument(
+    'page', required=True, type=int, default=1, location='args', help='The page to fetch')
+resources_parser.add_argument(
+    'page_size', required=True, type=int, default=20, location='args',
+    help='The page size to fetch')
+
+common_doc = {
+    'params': {'dataset': 'The dataset ID or slug'}
+}
+
+
+dataset_fields = apiv2.model('Dataset', {
```

Comment:
Is it possible to inherit from apiv1 and override the fields we want?

Reply:
Does not seem to work.

Reply:
Let's keep it this way for now, we'll iterate on this later, when we extend apiv2.